src/nvc.rs

```diff
@@ -459,7 +573,23 @@ impl Nvc {
                         }
                         OPCODE_SYSTEM_REGISTER_ID_PSW => self.set_reg_psw(value),
                         OPCODE_SYSTEM_REGISTER_ID_CHCW => {
-                            logln!("WARNING: ldsr chcw not yet implemented (value: 0x{:08x})", value);
```

Comment:
nit: I think we should still leave this log msg in here (as "not fully implemented" though), even though it adds extra info with the other messages later. This is meant to be both information for the user, as well as a note for the developers that this behavior may not be correct.